build/lib/buildkit.sh

```diff
@@ -45,5 +45,15 @@ if [ -f "/buildkit.sh" ]; then
     (exit $s)
 else
     # skip retry when running locally
-    $CMD "$@"
+    log_file=$(mktemp);
+    if ! $CMD "$@" 2>&1 | tee $log_file; then        
+        if grep "blobs/uploads/\": EOF" $log_file > /dev/null 2>&1; then
```

Comment:
```suggestion
        if grep -q "blobs/uploads/\": EOF" $log_file ; then
```
Just a little cleaner.